packages/eslint-config-next/rules/import.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
         'import/no-extraneous-dependencies': 0,
       },
     },
+    // Next.js default exports
+    {
+      files: ['app/**', 'src/pages/**', 'pages/**', 'next.config.{js,mjs}'],
```

Comment:
For app shouldn't this only be `app/**/{page,layout,loading,not-found,error,global-error,template,deafult,index,_app,_document}.tsx` and `app/**/route.ts`? As per [the next doc's file name conventions](https://nextjs.org/docs/app/building-your-application/routing#file-conventions)?

Or does that not cover leaf segments 🤔 

I'm fine with it being generalized, just trying to prevent confusion from non-next used files (since generally we have component files using named exports and we will loose that for any component files in app router folder)

Reply:
I would say that trying to keep up with those isn't worth the tradeoff that comes with preventing default exports in other files.